docs/tuning_fledge.rst

```diff
@@ -933,3 +934,47 @@ Care should be taken when using performance counters, as with almost any system
 
   Performance counters can be a very useful tool when tuning or debugging Fledge systems, but should **never** be left on during production use.
 
+
+Resource Limit Configuration of South Services
+==============================================
+
+Fledge includes a *Resource Limit* configuration for South Services, offering controls to manage resource usage and buffering effectively, thereby preventing excessive memory consumption or system overload due to unbounded buffering.
+
++---------------------------------+
+| |resource_limit_south_advanced| |
++---------------------------------+
+
+The following parameters are available for configuration:
+
+  - **South Service Buffering** : Defines whether the buffering for South Services is unlimited or capped. If set to `"Limited"`, additional configuration options become applicable.  
+
+  - **South Service Buffer Size** : Specifies the maximum number of readings that can be buffered in the South Service. This setting is only valid when the *South Service Buffering* option is set to `"Limited"`.  
+
+  - **Discard Policy** : Determines the policy for discarding readings when the buffer limit is reached. This setting is only valid when the *South Service Buffering* option is set to `"Limited"`.  
+
+     - **Discard Oldest**: Removes the oldest readings to keep the buffer size within the limit.  
+
+     - **Reduce Fidelity**: Reduces the fidelity of buffered readings by discarding every second reading, starting from the oldest. This policy tracks the next reading to discard to avoid repeated reduction of fidelity for the same data.  
+     
+     - **Discard Newest**: Discards the newest readings to maintain the buffer size.  
+
+Access Control
+--------------
+Only users with administrative privileges can modify the **Resource Limit** configuration items.
+
+Dynamic Configuration Updates
```

Comment:
No need to say this. All Fledge configuration is Dynamic, with a few exceptions that should already be documented. Saying this will mislead people into thinking they need to do restarts for other things that do not require it.